src/foremast/elb/create_elb.py

```diff
@@ -274,3 +277,17 @@ def configure_attributes(self, json_data):
             LOG.debug('Load Balancer Attributes:\n%s', pformat(load_balancer_attributes))
             elbclient.modify_load_balancer_attributes(
                 LoadBalancerName=self.app, LoadBalancerAttributes=load_balancer_attributes)
+
+    def add_elb_tags(self):
```

Comment:
Could this be moved to a utility function such as was done for s3?
https://github.com/foremast/foremast/blob/ee3b2290a6ad2d4a22c384237a17997e16a00d8b/src/foremast/s3/s3apps.py#L251